streamer/parser/a11y-metadata-parsing.md

```diff
@@ -0,0 +1,271 @@
+- `displayTransformability/background-color`
+- `highContrastAudio`
+- `highContrastAudio/noBackground`
+- `highContrastAudio/reducedBackground`
+- `highContrastAudio/switchableBackground`
+- `highContrastDisplay`
+- `index`
+- `largePrint`
+- `latex`
+- `longDescription`
+- `MathML`
+- `none`
+- `printPageNumbers`
+- `readingOrder`
+- `rubyAnnotations`
+- `signLanguage`
+- `structuralNavigation`
+- `synchronizedAudioText`
+- `tableOfContents`
+- `taggedPDF`
+- `tactileGraphic`
+- `tactileObject`
+- `timingControl`
+- `transcript`
+- `ttsMarkup`
+- `unlocked`
+
+Note the enumerated values are actually open-ended, due to the possible `displayTransformability` suffixes which map to CSS rules.
+
+### EPUB 2.x
+
+The array is created from the `meta` elements whose `name` attribute has the value `schema:accessibilityFeature`. The value of their `content` attribute is pushed to the array.
+
+### EPUB 3.X
+
+The array is created from the `meta` elements whose `property` attribute has the value `schema:accessibilityFeature`. Their value is pushed to the array.
+
+## AcessibilityHazard
+
+`accessibilityHazard` is a key whose value is an array of strings.
+
+It is expected in a conformant EPUB publication ([EPUB Accessibility 1.0](https://www.w3.org/Submission/epub-a11y/#sec-disc-package)).
+
+At the time of writing, known values are:
+
+- `flashingHazard`
+- `noFlashingHazard`
+- `motionSimulationHazard`
+- `noMotionSimulationHazard`
+- `soundHazard`
+- `noSoundHazard`
+- `unknown`
+- `none`
+
+### EPUB 2.x
+
+The array is created from the `meta` elements whose `name` attribute has the value `schema:accessibilityHazard`. The value of their `content` attribute is pushed to the array.
+
+### EPUB 3.X
+
+The array is created from the `meta` elements whose `property` attribute has the value `schema:accessibilityHazard`. Their value is pushed to the array.
+
+## AccessibilityControl
+
+`accessibilityControl` is a key whose value is an array of strings.
+
+At the time of writing, known values are:
+
+- `fullAudioControl`
+- `fullKeyboardControl`
+- `fullMouseControl`
+- `fullSwitchControl`
+- `fullTouchControl`
+- `fullVideoControl`
+- `fullVoiceControl`
+
+### EPUB 2.x
+
+The array is created from the `meta` elements whose `property` attribute has the value `schema:accessibilityControl`. The value of their `content` attribute is pushed to the array.
+
+### EPUB 3.X
+
+The array is created from the `meta` elements whose `property` attribute has the value `schema:accessibilityControl`. Their value is pushed to the array.
+
+## AccessibilityAPI
+
+`accessibilityAPI` is a key whose value is an array of strings.
+
+At the time of writing, known values are:
+
+- `ARIA`
+- `AndroidAccessibility`
+- `ATK`
+- `AT-SPI`
+- `BlackberryAccessibility`
+- `IAccessible2`
+- `iOSAccessibility`
+- `JavaAccessibility`
+- `MacOSXAccessibility`
+- `MSAA`
+- `UIAutomation`
+
+Currently, only the `ARIA` value might be trustworthy in the EPUB context.
```

Comment:
I'm not sure `trustworthy` is an intrinsic property of the term `ARIA` :)
Perhaps this instead:
"Currently, only the `ARIA` is known to be relevant in the EPUB context."